src/index.html

```diff
@@ -2,12 +2,12 @@
 layout: default
 ---
 
+{% include alert.html display="d-block" %}
```

Comment:
Pass in `display="d-none"` here to hide https://getbootstrap.com/docs/5.2/utilities/display/#notation